contributor_docs/README.md

```diff
@@ -17,8 +17,8 @@ Next, we are currently prioritizing work that expands access (inclusion and acce
 
 # Get Started
 Now you are ready to start contributing to p5.js! There are many ways to get started with contributing to p5.js and many reasons to do so. For the purpose of this documentation, we will split contributions roughly into two categories.
-- Contributions that directly deals with the source code (including documentation)
-- Contributions that directly deals with the source code very little or not at all
+- Source code contribution (including documentation)
+- Non-source code contribution
 
```

Comment:
I change the two category title to the chapter titles. If it's making things easier, we can remove 'including documentation' in 'Source code contribution,' and include it in 'Non-source code contribution' chapter.

Reply:
I like keeping "including documentation". And would also like to add to 'non code contributions' (see below) so it's more clear where to find out about that.